recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set version = "2022.07.19" %}
-{% set sha256sum = "6ed95025fba2aef0ce7b647607225745624497f876d74ef6ec22b26e73e9de77" %}
+{% set version = "2022.10.11" %}
+{% set sha256sum = "2cff03f9efdaf52626bd1b451d700605dc1ea000c5da56bd0fc59f8f43071040" %}
 
 {% set reldate = "{:d}-{:02d}-{:02d}".format(*(version.split(".") | map("int"))) %}
 
```

Comment:
As highlighted by the linter the source could be changed to https://curl.se/ca/cacert-{{ reldate }}.pem